tailwind.config.ts

```diff
@@ -82,20 +82,28 @@ export default {
     // https://www.figma.com/design/6M2LrpSCcB0thlFDaQAI2J/cx_jod_client?node-id=2217-5279
     fontSize: {
       'heading-1': ['36px', { fontWeight: '600', lineHeight: '50px' }], // Headline 1
+      'heading-1-mobile': ['32px', { fontWeight: '600', lineHeight: '44px' }], // Headline 1, mobile
       'heading-2': ['24px', { fontWeight: '600', lineHeight: '34px' }], // Headline 2
+      'heading-2-mobile': ['22px', { fontWeight: '600', lineHeight: '30px' }], // Headline 2, mobile
       'heading-3': ['18px', { fontWeight: '600', lineHeight: '27px' }], // Headline 3
+      'heading-3-mobile': ['16px', { fontWeight: '600', lineHeight: '24px' }], // Headline 3, mobile
       'heading-4': ['16px', { fontWeight: '600', lineHeight: '24px' }], // Headline 4
+      'heading-4-mobile': ['14px', { fontWeight: '600', lineHeight: '24px' }], // Headline 4, mobile
       'heading-5': ['16px', { fontWeight: '400', lineHeight: '24px' }], // Headline 5
-      'body-lg': ['18px', { fontWeight: '400', lineHeight: '26px' }], // Body L
-      'body-md': ['16px', { fontWeight: '400', lineHeight: '24px' }], // Body M
-      'body-sm': ['14px', { fontWeight: '400', lineHeight: '20px' }], // Body S
+      'body-lg': ['18px', { fontWeight: '400', lineHeight: '26px' }], // Body L (ingressi)
+      'body-lg-mobile': ['16px', { fontWeight: '400', lineHeight: '24px' }], // Body L, mobile (ingressi)
+      'body-md': ['16px', { fontWeight: '400', lineHeight: '24px' }], // Body M (leipäteksti)
+      'body-md-mobile': ['15px', { fontWeight: '400', lineHeight: '23px' }], // Body M, mobile (leipäteksti)
+      'body-sm': ['14px', { fontWeight: '400', lineHeight: '20px' }], // Body S (pieniteksti)
+      'body-sm-mobile': ['13px', { fontWeight: '400', lineHeight: '19px' }], // Body S, mobile (pieniteksti)
       'body-xs': ['12px', { fontWeight: '400', lineHeight: '16px' }], // Body XS
       'form-label': ['16px', { fontWeight: '700', lineHeight: '24px' }], // Form label
       help: ['16px', { fontWeight: '400', lineHeight: '24px' }], // Help, hint
       'button-lg': ['24px', { fontWeight: '600', lineHeight: '34px' }], // Button L
```

Comment:
At the time of writing there was not information available if there is mobile version of the `button-lg`. Not that this whole large button is used anywhere at the moment.

But if there would be and using the same logic it could be:

```ts
'button-lg-mobile': ['22px', { fontWeight: '600', lineHeight: '30px' }], // Button L, mobile
```